flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/LegacyTableSourceITCase.scala

```diff
@@ -347,7 +349,7 @@ class LegacyTableSourceITCase extends BatchTestBase {
          |)
        """.stripMargin
     tEnv.executeSql(ddl)
-    val resultPath = TEMPORARY_FOLDER.newFolder().getAbsolutePath
+    val resultPath = TempDirUtils.newFolder(BatchAbstractTestBase.tmpDir).getAbsolutePath
```

Comment:
nit: can you provide an utility method in BatchAbstractTestBase to wrap `TempDirUtils.newFolder(BatchAbstractTestBase.tmpDir)`